src/elfpy/simulators.py

```diff
@@ -235,34 +226,43 @@ def setup_pricing_and_market(self, override_dict=None):
             self.init_share_price = (1 + self.vault_apy[0]) ** self.init_vault_age
             if self.precision is not None:
                 self.init_share_price = np.around(self.init_share_price, self.precision)
-        self.set_pricing_model(self.pricing_model_name) # construct pricing model object
-        self.init_time_stretch = self.pricing_model.calc_time_stretch(self.init_pool_apy)
+        # setup pricing model
+        self.set_pricing_model(self.pricing_model_name)  # construct pricing model object
+        # setup market
+        time_stretch_constant = self.pricing_model.calc_time_stretch(self.init_pool_apy)
         init_reserves = self.pricing_model.calc_liquidity(
             self.target_liquidity,
             self.base_asset_price,
             self.init_pool_apy,
-            self.pool_duration,
-            self.init_time_stretch,
+            self.token_duration,
+            time_stretch_constant,
             self.init_share_price,  # u from YieldSpace w/ Yield Baring Vaults
             self.init_share_price,  # c from YieldSpace w/ Yield Baring Vaults
         )
         init_base_asset_reserves, init_token_asset_reserves = init_reserves[:2]
-        init_time_remaining = self.pricing_model.days_to_time_remaining(self.pool_duration, self.init_time_stretch)
         self.market = Market(
-            base_asset=init_base_asset_reserves,  # x
-            token_asset=init_token_asset_reserves,  # y
+            share_reserves=init_base_asset_reserves,  # x
```

Comment:
Nit: "x" => "z". It would also be good to leave a TODO here so that we remember to update from `init_base_asset_reserves` to another value.

Reply:
Added the TODO